golang/cosmos/x/vstorage/vstorage.go

```diff
@@ -61,6 +61,10 @@ func (sh vstorageHandler) Receive(cctx *vm.ControllerContext, str string) (ret s
 		keeper.SetStorageAndNotify(cctx.Context, msg.Path, msg.Value)
 		return "true", nil
 
+	case "setWithoutNotify":
+		keeper.SetStorage(cctx.Context, msg.Path, msg.Value)
```

Comment:
It's a bit unfortunate we have an inconsistent negation between the on the wire message and the underlying API, but I don't see a way to make this consistent without requiring extensive search/replace.

Reply:
I've made it consistent using `setterMethod`.  PTAL.

Reply:
I think my comment was regarding the naming difference between golang and js. E.g. `setWithoutNotify` -> `SetStorage` and `set` -> `SetStorageAndNotify`. It's a little jarring, but as I said, no big deal